Policies/github-username-policy.md

```diff
@@ -0,0 +1,24 @@
+---
+title: GitHub Username Policy
+productVersions:
+  dotcom: '*'
+englishOnly: true
+---
```

Comment:
Should there be a redirect-from in the frontmatter?

Reply:
yes! just added